docs/en/404.md

```diff
@@ -0,0 +1,10 @@
+---
+title: 404: Page not found
+---
+
+<div class="p-strip is-deep">
+  <div class="row">
+    <h1>404: Page not found</h1>
+    <p class="p-heading--four" style="max-width: none">Sorry, we can't find the page you're looking for.<br>If you think this is an error on our site, please <a class="p-link--external" href="https://github.com/canonical-websites/vanillaframework.io/issues/new">file a bug.</a></p>
```

Comment:
I understand why you're overriding the `max-width` here - that wrapped "file \n a bug" looks rather naff. But could you still constrain it a bit? Maybe `35em`? 'cos at the moment, if someone were to make that content a bit longer, it would stretch the entire width of the screen.

```suggestion
    <p class="p-heading--four" style="max-width: 35em">Sorry, we can't find the page you're looking for.<br>If you think this is an error on our site, please <a class="p-link--external" href="https://github.com/canonical-websites/vanillaframework.io/issues/new">file a bug.</a></p>
```